src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+import calculator.constants.ErrorMessage;
+import calculator.utils.StringSplitter;
+import calculator.utils.StringToIntegerParser;
+import calculator.utils.StringUtils;
+import java.util.Arrays;
+import java.util.List;
+
+public class StringAddCalculator {
+
+    private static final Integer DEFAULT_RESULT = 0;
+    private static final String POSITIVE_NUMBER_REGEX = "^[0-9]+$";
+
+    public static Integer splitAndSum(String text){
+        if(StringUtils.isEmpty(text)){
```

Comment:
사실 컨벤션은 같이 협업하는 사람들끼리 정하거나 회사별로 다르긴 할텐데요 
이번 과정 만큼은 구글 컨벤션으로 한번 연습해보시기를 추천 드립니다 😄 🙏 
```suggestion
        if (StringUtils.isEmpty(text)) {
```
> https://google.github.io/styleguide/javaguide.html#s4-formatting

Reply:
이 부분을 아애 신경을 쓰지 못한 것 같습니다. 사실 과제 내용에도 있었는데 
미션 자체만 집중한 것 같습니다 ㅠ 

조금 더 꼼꼼히 살펴보고 이번 기회로 구글 컨벤션을 명확히 이해하고 사용할 수 있도록 노력하겠습니다. :)